AWSAuthSDK/Tests/AWSMobileClientTests/AWSMobileClientHostedUITests.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  AWSMobileClientHostedUITests.swift
+//  AWSMobileClientTests
+//
+
+import XCTest
+
+@testable import AWSMobileClient
+import AWSAuthCore
+import AWSCognitoIdentityProvider
+
+/// AWSMobileClient tests related to HostedUIOptions
+class AWSMobileClientHostedUITests: AWSMobileClientTestBase {
+    
+    static var scopes = ["openid", "email"]
+    
+    override class func setUp() {
+        // signs out the user
+        super.setUp()
+        XCTAssertNil(AWSMobileClient.default().scopes)
+        
+        // assuming successful signin using custom scopes with HostedUIOptions
+        // set custom scopes in keychain
+        AWSMobileClientHostedUITests.setHostedUIScopesInKeychain()
+    }
+    
+    static func setHostedUIScopesInKeychain() {
```

Comment:
This looks like another test, so instead of doing this in setup and a static function, we can re-write as a test.